test/index.ts

```diff
@@ -336,6 +361,18 @@ describe('.getToken()', () => {
     );
   });
 
+  it('should not use cached token if expiring soon', async () => {
+    const gtoken = new GoogleToken(TESTDATA);
+    gtoken.rawToken = {
+      access_token: 'mytoken',
+    };
+    gtoken.expiresAt = new Date().getTime() + 4 * 60 * 1000;
+    const fakeToken = 'abc123';
+    const scope = createGetTokenMock(200, {access_token: fakeToken});
+    const token = await gtoken.getToken({forceRefresh: true});
```

Comment:
If possible, let's get rid of some of the warnings. Can you call `scope.done()` to ensure that the mock finished, as an example?